instances/widgets.treasury-factory.near/widget/pages/lockup/Table.jsx

```diff
@@ -0,0 +1,454 @@
+          props={{
+            instance,
+            amountWithoutDecimals: item.kind.FunctionCall.actions[0].deposit,
+          }}
+        />
+      </td>
+      <td className={isVisible("Start Date")}>
+        <Widget
+          src={`${REPL_BASE_DEPLOYMENT_ACCOUNT}/widget/components.Date`}
+          props={{ timestamp: startTimestamp }}
+        />
+      </td>
+      <td className={isVisible("End Date")}>
+        <Widget
+          src={`${REPL_BASE_DEPLOYMENT_ACCOUNT}/widget/components.Date`}
+          props={{
+            timestamp:
+              vestingSchedule.end_timestamp ??
+              parseInt(startTimestamp) + parseInt(args.release_duration),
+          }}
+        />
+      </td>
+      {allowLockupCancellation && (
+        <>
+          <td className={isVisible("Cliff Date")}>
+            {vestingSchedule.cliff_timestamp ? (
+              <Widget
+                src={`${REPL_BASE_DEPLOYMENT_ACCOUNT}/widget/components.Date`}
+                props={{ timestamp: vestingSchedule.cliff_timestamp }}
+              />
+            ) : (
+              "-"
+            )}
+          </td>
+          <td className={isVisible("Allow Cancellation")}>
+            {!!vestingSchedule ? "Yes" : "No"}
+          </td>
+        </>
+      )}
+      <td className={isVisible("Allow Staking")}>
+        {args.whitelist_account_id === "lockup-no-whitelist.near"
+          ? "No"
+          : "Yes"}
+      </td>
+      <td className={isVisible("Required Votes") + " text-center"}>
+        {requiredVotes ?? "-"}
+      </td>
+      {isPendingRequests && (
+        <td className={isVisible("Votes") + " text-center"}>
+          <Widget
+            src={`${REPL_BASE_DEPLOYMENT_ACCOUNT}/widget/components.Votes`}
+            props={{
+              votes: item.votes,
+              requiredVotes,
+            }}
+          />
+        </td>
+      )}
+      <td className={isVisible("Approvers") + " text-center"}>
+        <Widget
+          src={`${REPL_BASE_DEPLOYMENT_ACCOUNT}/widget/components.Approvers`}
+          props={{
+            votes: item.votes,
+            approversGroup: functionCallApproversGroup?.approverAccounts,
+            instance,
+          }}
+        />
+      </td>
+      {isPendingRequests && (hasVotingPermission || hasDeletePermission) && (
+        <td className={isVisible("Actions") + " text-center"}>
+          <Widget
+            src={`${REPL_BASE_DEPLOYMENT_ACCOUNT}/widget/components.VoteActions`}
+            props={{
+              instance,
+              votes: item.votes,
+              proposalId: item.id,
+              hasDeletePermission,
+              hasVotingPermission,
+              proposalCreator: item.proposer,
+              tokensBalance: [
+                {
+                  contract: "near",
+                  amount: Big(lockupNearBalances?.available ?? 0).toFixed(2),
+                },
+              ],
```

Comment:
we don't need to check for dao balance, since we don't allow user to create the request if they don't have sufficient balance, I hope? or do we?
if yes, please use ` avoidCheckForBalance: true,`
if not, I have changed the parameter, it needs to passed like [this](https://github.com/NEAR-DevHub/neardevhub-treasury-dashboard/blob/staging/instances/widgets.treasury-factory.near/widget/pages/payments/Table.jsx#L516)